### PR TITLE
feat: enable rc branch release flow

### PR DESCRIPTION
**Pull request by artemchernyak-outreach**
enabe rc branch release flow

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,8 +11,5 @@
     "prettier": "^2.3.2",
     "semantic-release": "^19.0.0",
     "semver": "^7.3.5"
-  },
-  "dependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.3"
```
Comment (jaredallard) [inline package.json:16]: We need this

Comment (jaredallard) [reply package.json:16]: Ah, yeah, so we do need a block here, but `package.json` doesn't actually support blocks. I want to `fromJSON` this and keep user's changes by basically doing a merge but I haven't gotten around to it yet :(

Comment (artemchernyak-outreach) [reply package.json]: Oh yeah... I forgot about that json annoyance. Oh well, I just added it back as a short term fix. Maybe we can get the `fromJSON` loading working by next release.

File: service.yaml [hunk 2]
```diff
@@ -11,6 +11,7 @@ metrics: datadog
 releaseType: ""
 releaseOptions:
   allowPrereleases: true
+  prereleasesBranch: ""
```
Comment (jaredallard) [inline service.yaml:14]: Should this have a value? 🤔

Comment (artemchernyak-outreach) [reply service.yaml]: Sorry about that, looks like the PR ran with RC versions. All fixed up now.

**Review by jaredallard — APPROVED** [review]

**Pull request merged**